docs/plugins.rst

```diff
@@ -70,6 +86,33 @@ which you map the signals to your plugin logic. Let's take a simple example::
     your ``register`` callable or they will be garbage-collected before the
     signal is emitted.
 
+Namespace Plugin structure
+--------------------------
+
+Namespace plugins must adhere to a certain structure in order to function
+properly. They need to installable (i.e. contain ``setup.py`` or equivalent)
+and have a folder structure as follows::
+
+    myplugin
+    ├── pelican
+    │   └── plugins
+    │       └── myplugin
+    │           ├── __init__.py
+    │           └── ...
+    ├── ...
+    └── setup.py
+
+It is crucial that ``pelican`` or ``pelican/plugins`` folder **should not**
```

Comment:
`**should not**` → `**not**` (already implied via "crucial")

Reply:
I would probably remove the word "should" altogether, but that's admittedly a very minor nitpick. 😅

Reply:
Done :)